tests/Sabre/Xml/ServiceTest.php

```diff
@@ -123,6 +125,40 @@ function testExpect() {
         );
     }
 
+    /**
+     * @expectedException \Sabre\Xml\LibXMLException
```

Comment:
The support for this annotation will be removed with version 9 of PHPUnit. This would not be a problem if this package did not use the "death star version constraint" on PHPUnit.

* https://github.com/sebastianbergmann/phpunit/issues/3333
* https://thephp.cc/news/2017/02/death-star-version-constraint

My suggestion: use the [`expectException`](https://phpunit.readthedocs.io/en/7.4/writing-tests-for-phpunit.html#testing-exceptions) method instead and define a version for PHPUnit in the Composer configuration – version 8 is around the corner.

Reply:
at first we should get rid of the "death star version constraint".

changes to our unit tests itself are welcome via separate pull requets

Reply:
@staabm 
> thx for noticing.

Right, this was only a notice and should be done in a separate PR.

Reply:
This is the 1.5 branch - it is on pure maintenance mode. No idea if we should put any further man power into this.

Reply:
> This is the 1.5 branch - it is on pure maintenance mode. No idea if we should put any further man power into this.

the mentioned issues should be fixed on master.

in 1.5 I would only fix the "death star version constraint"